.github/workflows/ci.yml

```
@@ -1,42 +1,30 @@
-name: CI
-
-on:
-  pull_request:
-    branches: [main]
-  push:
-    branches: [main]
-
+name: ci
+on: [push, pull_request]
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)